lib/blockchain.go

```diff
@@ -1509,7 +1509,9 @@ func (bc *Blockchain) _validateOrphanBlock(desoBlock *MsgDeSoBlock) error {
 	if err != nil {
 		return fmt.Errorf("_validateOrphanBlock: Could not serialize block")
 	}
-	if uint64(len(serializedBlock)) > bc.params.MaxBlockSizeBytes {
+	// TODO: seems like we don't use this function w/ PoS so it's safe to
+	// leave this as a direct access to MaxBlockSizeBytesPoW.
```

Comment:
Yeah, this is fine as-is. Would it make sense to rename the above function to `_validateOrhpahBlockPoW`? NBD either way.

Reply:
done!